Makefile

```
@@ -62,36 +60,26 @@ push_translations: ## push translations to transifex
 
 check_translations_up_to_date: extract_translations compile_translations dummy_translations detect_changed_source_translations ## extract, compile, and check if translation files are up-to-date
 
-install_firefox:
-	@mkdir -p test_helpers
-	@test -f ./test_helpers/firefox/firefox && echo "Firefox already installed." || \
-	(cd test_helpers && \
-	wget -N "https://archive.mozilla.org/pub/firefox/releases/$(FIREFOX_VERSION)/linux-x86_64/en-US/firefox-$(FIREFOX_VERSION).tar.bz2" && \
-	tar -xjf firefox-$(FIREFOX_VERSION).tar.bz2)
-
 piptools: ## install pinned version of pip-compile and pip-sync
 	pip install -r requirements/pip.txt
 	pip install -r requirements/pip-tools.txt
 
 requirements: piptools  ## install test requirements locally
 	pip-sync requirements/ci.txt
 
-requirements_python: install_firefox piptools  ## install all requirements locally
+requirements_python: piptools  ## install all requirements locally
 	pip-sync requirements/dev.txt requirements/private.*
 
 test.quality: selfcheck ## run quality checkers on the codebase
 	tox -e quality
 
-test.python: ## run python unit and integration tests
-	PATH=test_helpers/firefox:$$PATH xvfb-run python run_tests.py $(TEST)
+test.python: ## run python unit tests
+	python run_tests.py $(TEST)
```

```suggestion
test.python: ## run python unit tests in the local virtualenv
	pytest --cov drag_and_drop_v2 $(TEST)
```

Then, let's remove `run_tests.py` and replace the example in the README with `make test.python TEST=tests/unit/test_basics.py::BasicTests::test_student_view_data`.